examples/simple/main.tf

```diff
@@ -0,0 +1,44 @@
+#
+# ECS Service Module
+#
+
+module "app_ecs_service" {
+  source = "../../"
+
+  name        = "terratest-simple"
```

Comment:
In the other one you updated, you used var rather than the name. 
https://github.com/trussworks/terraform-aws-config/commit/4644e7338a7b56e9e9f99a624f5ceae54a7bf863
I think that is a better practice than this as it is more generic, or if not that then just naming this something like "simple" rather than terratest-simple.
I'm pointing this out so we can have consistency.